README.md

```diff
@@ -23,7 +23,12 @@ To send to Intercom:
 - set `INTERCOM_API_KEY=123`
 - set `INTERCOM_APP_ID=456`
 
-[Mixpanel][mixpanel], [Drip][drip] and [Keen.io][keen.io] are probably going to be next.
+To send to Drip:
+
+- set `DRIP_ACCOUNT_ID=234` (found here: https://www.getdrip.com/3947613/settings/site under "3rd party integrations")
```

Comment:
The URL is specific to your account, so maybe change that number to `YOUR ACCOUNT ID HERE`...or not...not a big deal :)